pypeit/core/gui/object_find.py

```diff
@@ -0,0 +1,684 @@
+import os
```

Comment:
Please add a line or two describing this module.

Reply:
OK done!